client/me/privacy/do-not-sell.tsx

```diff
@@ -0,0 +1,25 @@
+import { Card } from '@automattic/components';
+import { ToggleControl } from '@wordpress/components';
+import { useDoNotSellContent } from 'calypso/blocks/do-not-sell-dialog/use-do-not-sell-content';
+import SectionHeader from 'calypso/components/section-header';
+import { useDoNotSell } from 'calypso/lib/analytics/utils';
+
+export const DoNotSellSetting = () => {
+	const { shouldSeeDoNotSell, isDoNotSell, onSetDoNotSell } = useDoNotSell();
+	const { title, toggleLabel, longDescription } = useDoNotSellContent();
+
+	if ( ! shouldSeeDoNotSell ) {
+		return null;
+	}
+
+	return (
+		<>
+			<SectionHeader label={ title } />
+			<Card className="privacy__settings">
+				{ longDescription }
+				<hr />
```

Comment:
Ugh, I thought we don't do these kinds of separators anymore 😅 Can we just use two cards, and/or take advantage of the `CompactCard` component if necessary?

Reply:
I'm unfamiliar with `Card` and `CompactCard` components. However, I based my solution on the [component that exists](https://github.com/Automattic/wp-calypso/blob/add/implement-do-not-sell-dialog-on-public-pages/client/me/privacy/main.jsx#L104) on /me/privacy page already, as I wanted to achieve a similar effect:
<img width="1054" alt="CleanShot 2022-12-13 at 10 07 58@2x" src="https://user-images.githubusercontent.com/8419292/207384270-7f0934fd-d5c5-4e48-b952-3b9e77d4ed8c.png">

<img width="1056" alt="CleanShot 2022-12-13 at 10 08 13@2x" src="https://user-images.githubusercontent.com/8419292/207384343-d4e5e204-cd30-4bf3-9fb7-74362500745c.png">

Does it mean that this way of adding a separator between content and action buttons is outdated?

Reply:
I see the `<SectionHeader />` is using `<CompactCard />`. With certain modifications, I could also use it that way, however, it already modifies the design:
<img width="1063" alt="CleanShot 2022-12-13 at 10 15 53@2x" src="https://user-images.githubusercontent.com/8419292/207386362-8e14a42c-06b9-41fc-a235-c0f1b3f3e59a.png">

Reply:
I have to say I slightly prefer the modified version, but I'm not a designer, so that opinion doesn't worth much. Maybe we can ping someone?

Reply:
I don't mind at all 😉 This isn't a blocker, and is definitely something we can follow up with in another PR.

Reply:
Thanks for clarifying!